src/EditorFeatures/Core/Structure/StructureTag.cs

```diff
@@ -62,7 +62,18 @@ public StructureTag(AbstractStructureTaggerProvider tagProvider, BlockSpan block
         public bool IsImplementation { get; }
 
         public override int GetHashCode()
-            => throw ExceptionUtilities.Unreachable();
```

Comment:
This generally worries me about the other tag types where we did this as well -- seems like we should just be implementing it everywhere.

Reply:
ok.  i won't throw.  but i'm not going to impl with sensible semantics (since we weren't before).

Reply:
done.